GitUI/Strings.cs

```diff
@@ -42,6 +42,12 @@ internal sealed class Strings : Translate
         private readonly TranslationString _errorCaptionFailedDeleteFile = new TranslationString("Failed to delete file");
         private readonly TranslationString _errorCaptionFailedDeleteFolder = new TranslationString("Failed to delete directory");
 
+        private readonly TranslationString _noBranchTitle = new TranslationString("no branch");
+        private readonly TranslationString _directoryIsNotAValidRepositoryRemoveSelectedRepoCommand = new TranslationString("Remove the selected invalid repository");
```

Comment:
Strings in Strings.cs should have the intention to be shared, the names are far to specific here to allow that

Reply:
Suggestions?

In WinForms we place all resources in a single file, e.g. https://github.com/dotnet/winforms/blob/master/src/System.Windows.Forms/src/Resources/SR.resx

Reply:
Actually a good point, we need to update the dashboard because it has the same strings.